### ISSUE TITLE
chromdriver version and headless chrome

### ISSUE THREAD

**Issue by bricas** [john-doherty/selenium-cucumber-js#44]
We're doing some experimenting with headless chrome in place of phantomjs for our testing.

It seems headless chrome needs a pretty recent version of chromedriver. This package currently only gets chromedriver 2.28, and the "[getting started with headless chrome](https://developers.google.com/web/updates/2017/04/headless-chrome#drivers)" page recommends at least version 2.32 (latest: 2.33)

Any chance for an update?

**Comment by john-doherty** [comment]
Hi @bricas, I don't have time at the moment but it would be great if you could fork the repo, make the change and issue a pull request as part of your experimentation work :)

**Comment by bricas** [comment]
Hey @john-doherty, I'm closing this ticket. Although I'd still recommend updating the dep version, here's what I've found:

- There tends to be a conflict between the chromedriver and electron-chromedriver packages as they both want to put their own chromedriver in the path
- The chromium package for alpine linux also has a chromedriver package, so i can remove executable downloaded by npm and my system's chromedriver will be used (which i know will work with the version of chromium installed)
- I can create my own driver file (which your package will happily load) and pass the necessary options to get chrome headless to run

All of that to say I've worked around the issues I _thought_ I was having. :)

Cheers.
